konduit-serving-python/src/main/java/ai/konduit/serving/executioner/PythonExecutioner.java

```diff
@@ -197,7 +197,8 @@ public static synchronized void resetInterpreter(String interpreterName) {
         log.info("Resetting interpreter " + interpreterName);
         String oldInterpreter = currentInterpreter;
         setInterpreter(interpreterName);
-        exec(interpreterName);
+        exec("pass");
```

Comment:
May want to comment this here. I know what "pass" is but it might not be immediately obvious what the side effects are.